apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -0,0 +1,90 @@
+import { cc, shape } from '@cube-creator/core/namespace'
+import { hydra, rdfs, sh, dcat, dcterms, schema } from '@tpluscode/rdf-ns-builders'
+import { turtle } from '@tpluscode/rdf-string'
+
+export const DatasetShape = turtle`
+${shape('dataset/edit-metadata')} {
+  ${shape('dataset/edit-metadata')} a ${sh.NodeShape}, ${hydra.Resource} ;
+    ${sh.targetClass} ${dcat.DataSet} ;
+    ${rdfs.label} "Cube Metadata" ;
+    ${sh.property} [
+      ${sh.name} "Title" ;
+      ${sh.path} ${dcterms.title} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.minLength} 1 ;
+      ${sh.order} 10 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Description" ;
+      ${sh.path} ${dcterms.comment} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.minLength} 1 ;
+      ${sh.order} 20 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Version" ;
+      ${sh.path} ${schema.version} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.minLength} 1 ;
+      ${sh.order} 30 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Publish on Swiss Open Data" ;
+      ${sh.path} ${cc.PublishOnOpendata} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.datatype} xsd:boolean
+      ${sh.order} 40 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Publish on Swiss Open Data" ;
+      ${sh.path} ${cc.PublishOnOpendata} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.datatype} xsd:boolean
+      ${sh.order} 50 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Publish on Visualize" ;
+      ${sh.path} ${dcterms.publishOnVisualize} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.datatype} xsd:boolean
+      ${sh.order} 60 ;
+    ] ;
+    ${sh.property} [
+        ${sh.name} "Contact Point" ;
+        ${sh.path} ${dcat.contactPoint} ;
+        ${sh.minCount} 1 ;
+        ${sh.maxCount} 1 ;
+        ${sh.minLength} 1 ;
+        ${sh.order} 70 ;
+      ] ;
+    ${sh.property} [
+      ${sh.name} "Tags" ;
+      ${sh.path} ${dcat.keyword} ;
+      ${sh.minLength} 1 ;
+      ${sh.order} 80 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Creation Date" ;
+      ${sh.path} ${dcterms.issued} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.datatype} xsd:date
+      ${sh.order} 90 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Last Update" ;
+      ${sh.path} ${dcterms.modified} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.datatype} xsd:date
+      ${sh.order} 100 ;
+    ] ;    
```

Comment:
I expect the dates to be automatically updated. The creation (I think there is `dcterm:created` by the way) would be set only once and `modified` should just update when making any changes

Reply:
Daniel used consequently issued in his document...

Reply:
> Daniel used consequently issued in his document...

I don't understand

> I was wondering if the user should be able to overwrite it

Most certainly not. Isn't the update simply the moment in time when it happens? Why would the users set it themselves?

Reply:
Document of Dani with an example from Adrian? <https://gitlab.ldbar.ch/bafu/umweltdatenkiosk-planning/-/wikis/Dataset-Metadata#draft>

Since this date is displayed to the end users on opendata etc. I would be not surprised if somebody wants to adapt them. But I am fine with the fact, that they could be hidden or read only.

Reply:
Ok, interesting point. Let's keep them then.